### DISCUSSION TITLE
Welcome to vedo Discussions!

### DISCUSSION THREAD

**Discussion by marcomusy** [marcomusy/vedo#261] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZiguoAtGitHub** [comment]
Hi all,
I am a civil engineer. You can call me Ziguo.🙂
I often do some numerical analysis work using finite element softwares  such as Ansys, Abaqus and so on. Sometimes I need some better ways for analysis results showing, including figures and animations. 
Vedo is a wonderful and helpful tool to get what I want! Thanks Marco for your great job and generous sharing spirit!
I only have several years' experience on Python and several months on Vedo. I love programming and hope to get and give ( I try) helps with this community.

**Comment by marcomusy** [comment]
Thanks Ziguo for your feedback!
